src/Compilers/Shared/GlobalAssemblyCacheHelpers/MonoGlobalAssemblyCache.cs

```diff
@@ -17,54 +17,48 @@ namespace Microsoft.CodeAnalysis
     /// </summary>
     internal sealed class MonoGlobalAssemblyCache : GlobalAssemblyCache
     {
-        public static readonly ImmutableArray<string> RootLocations;
+        private static readonly string CorlibDirectory;
+        private static readonly string GacDirectory;
 
         static MonoGlobalAssemblyCache()
         {
-            RootLocations = ImmutableArray.Create(GetMonoCachePath());
-        }
+            var corlibAssemblyFile = typeof(object).GetTypeInfo().Assembly.Location;
```

Comment:
Don't believe `GetTypeInfo` is necessary here.

Reply:
Other uses of `typeof()` to get at the type's assembly are going through `GetTypeInfo`. I also don't believe it should be necessary anymore, but I was trying to stay consistent. No opinion here (the other existing just below this one is `typeof(Uri).GetTypeInfo().Assembly...` to get at `System.dll`. I assume all of these are leftovers from pre-netstandard2.0 days?